bin/node-template/runtime/src/lib.rs

```diff
@@ -203,10 +203,15 @@ impl frame_system::Config for Runtime {
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
 }
 
+parameter_types! {
+	pub const AllowMultipleBlocksPerSlot: bool = false;
+}
+
 impl pallet_aura::Config for Runtime {
 	type AuthorityId = AuraId;
 	type DisabledValidators = ();
 	type MaxAuthorities = ConstU32<32>;
+	type AllowMultipleBlocksPerSlot = AllowMultipleBlocksPerSlot;
```

Comment:
Would it be possible and cleaner to achieve this with an associated constant? 

```suggestion
impl pallet_aura::Config for Runtime {
	type AuthorityId = AuraId;
	type DisabledValidators = ();
	type MaxAuthorities = ConstU32<32>;
	type AllowMultipleBlocksPerSlot = ConstBool<false>;
```

Reply:
Yes, will update